src/App.js

```diff
@@ -1,10 +1,79 @@
-import logo from './logo.svg';
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import { ToastContainer } from 'react-toastify';
+import { useSelector } from 'react-redux';
+import ReservationsList from './components/reservation/ReservationList';
+import Protected from './components/ProtectedRoute';
+import Home from './pages/Index';
+import Login from './pages/Login';
+import Register from './pages/Register';
+import AddYacht from './components/AddYacht';
+import Nav from './components/Nav';
+import DeleteYacht from './components/DeleteYacht';
+import 'react-toastify/dist/ReactToastify.css';
+import Reserve from './components/reservation/Index';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this main App component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪